src/content_delegate.js

```diff
@@ -180,6 +180,27 @@ ContentDelegate.prototype.findAndStorePacerDocIds = function () {
   });
 };
 
+// If this is a docket query page, add RECAP email advertisement.
+ContentDelegate.prototype.addRecapEmailAdvertisement = async function () {
+  if (!(PACER.isIQueryReportUrl(this.url) || PACER.isManageAccountPage(this.url))) { return; };
+  let form; 
+
+  if (!PACER.hasFilingCooking(document.cookie)){ return; };
+   
+  if (PACER.isIQueryReportUrl(this.url)){
+    form = document.querySelector('form');
+    if (!document.querySelector('recap-email-banner')){
```

Comment:
Shouldn't this (and below) be:

```suggestion
    if (!document.querySelector('.recap-email-banner')){
```

Reply:
You're right!

changed. Thanks